bin/deploy-ux-testing.sh

```diff
@@ -29,6 +29,7 @@ rm -rf seeds/test
 rm seeds/shared/delete-everything.js
 rm seeds/development.js
 rm seeds/test.js
+npm ci
```

Comment:
Here's a place where the deploy behavior was dependent on the external environment - if `node_modules` was already populated, we'd get a vendored dependency deploy; otherwise, we wouldn't.  Not a huge fan of that kind of "accidental" behavior.  This way it's more explicit - we get a vendored deploy.

Reply:
👍